device/cuda/src/finding/kernels/apply_interaction.cuh

```diff
@@ -14,8 +14,9 @@
 namespace traccc::cuda::kernels {
 
 template <typename detector_t>
-__global__ void apply_interaction(
-    const finding_config cfg,
-    device::apply_interaction_payload<detector_t> payload);
+void apply_interaction(const dim3& grid_size, const dim3& block_size,
```

Comment:
These are now C\+\+ headers. Let's rename them to `.hpp`! Or do they still in turn include some `.cuh` files of their own? 😕